### PR TITLE
add functional switches and mirroring

### PR DESCRIPTION
**Pull request by kevinvangelder**
It's small, but it's what I had time for today.

### CODE REVIEW

File: timewarpscanme/src/Scancam.js [hunk 1]
```diff
@@ -40,11 +40,11 @@ export default function (props) {
 
   useEffect(() => {
     setupWebcam()
-  });
+  }, []);
```
Comment (kevinvangelder) [inline timewarpscanme/src/Scancam.js:43]: Without this, the webcam is initialized on every rerender.

File: timewarpscanme/src/Warper.js [hunk 2]
```diff
@@ -1,16 +1,29 @@
 // https://material-ui.com/components/switches/
+import { useState } from 'react'
 import Scancam from './Scancam'
-import { FormControlLabel, FormGroup, Switch } from '@material-ui/core';
+import { FormControlLabel, FormGroup, Switch } from '@material-ui/core'
```
Comment (kevinvangelder) [inline timewarpscanme/src/Warper.js:4]: Bad semicolon, bad!

Comment (GantMan) [reply timewarpscanme/src/Warper.js:4]: ahahhahah

**Review by GantMan — APPROVED** [review]
perfect!

**Pull request merged**